horovod/common/parameter_manager.cc

```diff
@@ -152,9 +127,15 @@ void ParameterManager::SetCycleTimeMs(double value, bool fixed) {
   joint_params_.SetValue(cycle_time_ms, value, fixed);
 }
 
-void ParameterManager::Update(const std::vector<std::string>& tensor_names, int64_t bytes) {
+/// Using the tensors information to update score and tune parameters.
```

Comment:
Move member function comments to the header files.

Reply:
Fixed.